README.md

```diff
@@ -1,36 +1,47 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# AI Dashboard Generator
 
-## Getting Started
+AI Dashboard Generator is an open-source dashboard generator based on Next.js and ChatGPT. The tool allows users to feed existing tables or datasets, and from there, it understands the intent of the table and generates a visual representation of the data.
```

Comment:
Instead of "ChatGPT" it should be "OpenAI's GPT3 technology". Since we are using the GPT3 API instead of the Chat.